src/libs/elektra/opmphm_vheap.c

```diff
@@ -0,0 +1,141 @@
+#include "kdbprivate.h"
+
+/**
+ * Allocates all space needed.
```

Comment:
Very vague description. Do you mean "Allocates vheap with size as specified by parameter?"